Sources/Common/Communication/Event.swift

```diff
@@ -82,6 +82,20 @@ public enum EventTypesRegistry {
 // Each event struct should have properties relevant to its specific use case.
 // They must include `storageId`, `params`, and any other relevant information.
 
+public struct ProfileAlreadyIdentifiedEvent: EventRepresentable {
```

Comment:
I created a new event to keep things clear but was wondering if we could use the already existing event struct `ProfileIdentifiedEvent` for the already identified users too since the functioning is same i.e to call the `setProfileIdentifier` method

`self.inAppProvider.setProfileIdentifier(event.identifier)` 🤔

Reply:
Yup, you are right, it can be resused.

Reply:
Done